src/members/grid-list-item.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { memberShape } from './model';
+
+export default function MembersGridListItem(props) {
+  const { memberInfo } = props;
+  return (
+    <div className="member">
+      <img
+        src={memberInfo.portret.childImageSharp.fluid.src}
+        alt={`${memberInfo.title}`}
```

Comment:
This could simply be `alt={memberInfo.title}`.

Reply:
fixed :)